syntax = "proto3";

package order;

service OrderService {
    rpc CreateOrder(CreateOrderRequest) returns (OrderResponse);
    rpc GetOrderStatus(OrderStatusRequest) returns (OrderStatusResponse);
}

message CreateOrderRequest {
    string user_id = 1;
    string product_id = 2;
    int32 quantity = 3;
}

message OrderResponse {
    string order_id = 1;
    string status = 2;
}

message OrderStatusRequest {
    string order_id = 1;
}

message OrderStatusResponse {
    string order_id = 1;
    string status = 2;
}
